exec/plugin-commands-script-runners/package.json

```
@@ -58,13 +59,15 @@
     "@pnpm/store-path": "workspace:*",
     "@pnpm/types": "workspace:*",
     "@zkochan/rimraf": "^2.1.2",
+    "didyoumean2": "^5.0.0",
     "execa": "npm:safe-execa@0.1.2",
     "p-limit": "^3.1.0",
     "path-exists": "^4.0.0",
     "path-name": "^1.0.0",
     "ramda": "npm:@pnpm/ramda@0.28.1",
     "realpath-missing": "^1.1.0",
     "render-help": "^1.0.3",
+    "which": "^3.0.0",
```

which has a security vulnerability that I have fixed.

```suggestion
    "which": "npm:@zkochan/which@^2.0.3",
```